src/Navbar/Navbar.js

```diff
@@ -1,14 +1,23 @@
 import './Navbar.scss'
 
-function Navbar() {
-  return (
-    <nav className="navbar navbar-expand border-bottom shadow-sm px-4 justify-content-md-start justify-content-center">
-        <a href="#" className="navbar-brand fw-bold m-0">
-          <span className="brand-accent">Food</span>
-          <span>Delivery</span>
-        </a>
-    </nav>
-  )
+function Navbar(props) {
```

Comment:
all your other components are defined using fat arrow syntax, you should change this one to match for consistency

Reply:
Done